### ISSUE TITLE
Costing display option

### ISSUE THREAD

**Issue by AhmadShamli** [cp6/my-idlers#110]
Hi,

It would be best if there is an option to display actual cost instead of average. This would better reflect what need to pay in weekly, monthly, or yearly.

Or and option to show both to show average cost.

Thanks.

**Comment by cp6** [comment]
Weekly, monthly and yearly costs are already shown

**Comment by AhmadShamli** [comment]
> Weekly, monthly and yearly costs are already shown

yes, but that was average. what I'm suggesting is an actual cost in calendar month,and year/s. So user can expect what amount they need to prepare/pay for the current/next month/year.
